testing-guidance/workflow-tests/physics/physics-ragdoll-workflows.md

```diff
@@ -33,14 +34,14 @@ Any project that has the following Gems enabled:
+| **Verify Atom Render Window UI clusters** | <ol><li>Click the filter button in the top right corner of the Skeleton Outliner and check the *Ragdoll joints and colliders* filter for easier navigation.</li><li>In the Skeleton Outliner select a bone that has both a collider, and a joint limit enabled (for example *L_wrist_INT* from the *Rin.fbx*).</li> <li>With the bone still selected deselect the *Has joint limit* in the Joint Limit component in the Inspector pane.</li> <li>In the Skeleton Outliner select a bone, right-click it, and select **Ragdoll → Remove Colliders** from the context menu.</li> <li>Select bones in the Skeleton Outliner that either have both, one of, or no collider and joint limit enabled.</li><li>In the Skeleton Outliner select a bone that has both a collider and a joint limit enabled (for example *L_wrist_INT* from the *Rin.fbx*). Select all buttons from both clusters in the Atom Render Window in order. For the *Automatic joint limit setup* button, the Animation Editor layout has to be changed to AnimGraph (from Physics) and your Motion Set (for example *MotionMatching.motionset*) has to be added under the Motion Sets → Motion Set Management by clicking the *Load motion set from file* button. You can enable *Ragdoll Colliders* and *Ragdoll Joint Limits* under the *eye* button dropdown menu in the Atom Render Window for more visual clarity.</li></ol> | <ul><li>Filters properly apply to relevant bones.</li><li>Two button clusters show up in the top-left corner of the Atom Render Window when relevant bones are selected. First cluster from the top has three buttons used for modifying collider translation, rotation, and dimensions. Second cluster from the top has five buttons used for adjusting joint limits.</li></br>![](images/bone_button_clusters.png) <br><li>Disabling/enabling colliders/*Has joint limit* property hides/shows their relevant button clusters.</li><li>Switching between different bones properly updates button clusters.</li><li>Manipulators that show up in the Atom Renderer Window are functional and their changes are retained after being deselected.</li><li>Adding a Motion Set allows the *Automatic joint limit setup* button to be used.</li></ul> |
```

Comment:
* `With the bone still selected deselect the *Has joint limit*` - Should be `With the bone still selected, deselect the *Has joint limit*`. (Added ,)

Reply:
Updated